### ISSUE TITLE
Set UID, GID of downloaded files.

### ISSUE THREAD

**Issue by ajurna** [Casvt/Kapowarr#35]
**Is your feature request related to a problem? Please describe.**
the container runs as root but i don't want to store my comics with root user and group.

**Describe the solution you'd like**
support for setting the uid and gid of the files downloaded.

**Additional context**
would also be nice to set the umask for the files as well. 644 might not be what people want.

**Comment by Casvt** [comment]
Duplicate of #33 ?

**Comment by ajurna** [comment]
nope, that's for its running context. while it could accomplish the same it's not exactly the same thing. this would allow you to continue running as root in the container but the file permissions would change.

**Comment by Berserkir-Wolf** [comment]
Sounds like wanting to support UMASK (like linuxserver images), which maps the container root to a local user with less-than-root access.
That about right, @ajurna ?

**Comment by Berserkir-Wolf** [comment]
The above change creates a container-local user (so only existing in the container) named kapowarr on startup of the container, with a PUID:PGID able to be defined in environment variables.
The user has sudo/root access in the container, but does not on the host system.

**Comment by ajurna** [comment]
That is correct

**Comment by ajurna** [comment]
although looking at the docs changes that are being brought in for #59 having a umask setting could still be useful.

**Comment by ignacio82** [comment]
+1
